src/Activity.js

```diff
@@ -51,6 +51,20 @@ returnAverageForWeek(week, activityData) {
     return [stepObj, this.user.friends[totalStepsPerFriend.indexOf(Math.max(...totalStepsPerFriend))]]
   }
 
+  returnFriendsStairsCount() {
```

Comment:
This looks like the same function as returnFriendsStepCount() in the activity class and could easily be combined into one function with another parameter for property.

Reply:
These seem to break the when merged into one function. It's fine as is and we can refractor them when everything is working together